static/admin/css/base.css

```diff
@@ -0,0 +1,847 @@
+/*
```

Comment:
I think someone did a `./manage.py collectstatic` ... You don't need to do this except if you are deploying the website. If you do, it adds all of the Django admin styles which are not needed because they are installed with Django for the test environment. Please `git rm` them from the repo.